src/app/pages/datasets/components/dataset-quotas/dataset-quotas-userlist/dataset-quotas-userlist.component.ts

```diff
@@ -116,7 +116,12 @@ export class DatasetQuotasUserlistComponent implements OnInit, AfterViewInit, On
       case 'used_percent':
         return `${Math.round(row[field] * 100) / 100}%`;
       case 'obj_used_percent':
-        return `${Math.round(row[field] * 100) / 100}%`;
+        if (row.obj_quota) {
+          return `${Math.round(row.obj_used / row.obj_quota * 100) / 100}%`;
+        }
+        return this.translate.instant('N/A');
```

Comment:
I would show dash `–` instead of N/A. N/A doesn't immediately sounds like no quota, and I may instead think that quota was not loaded or something.

Reply:
if quota is set as 0 then it will remove the quota